src/node.rs

```diff
@@ -11,6 +11,36 @@ use crate::sys::Box;
 use crate::sys::{new_map_with_capacity, ChildrenVec, Map, Vec};
 use core::sync::atomic::{AtomicUsize, Ordering};
 
+#[cfg(test)]
+use crate::style::{Dimension, Display, FlexDirection};
```

Comment:
The idiomatic way to do tests in rust is to make a new module tests which is only compiled with `cfg(test)` and which contains all tests and test specific imports. That way, all the test functions are not compiled by default:

```rust
#[cfg(test)]
mod tests {
    use crate::style::{Dimension, Display, FlexDirection};
    
    // Test functions
}
```

Reply:
As outlined in https://github.com/DioxusLabs/taffy/discussions/179, wouldn't that make it impossible to test internal and private behavior?

Reply:
I agree that moving them to `mod tests` is cleaner, and this is why I'm struggling a bit to understand exactly what is the best place for the tests to be (hence the discussion)

Reply:
Because `mod tests` is inside the same module, it should have access to the private functions.

Does [this post](https://users.rust-lang.org/t/testing-private-functions/41097/3) help?

Reply:
Ah, in that case that is much better 💯 Thanks! This was the missing piece :)

Reply:
Done :)